ubuntu/derived/standard/Dockerfile.template

```diff
@@ -23,14 +23,28 @@ RUN echo "deb [arch=amd64] http://apt-mo.trafficmanager.net/repos/dotnet-release
     # Python
     python \
     python3 \
-    # Java and related build tools
-    openjdk-$(OPENJDK_VERSION)-jdk \
-    ant \
-    ant-optional \
     # .NET Core SDK
     dotnet-dev-1.0.1 \
  && rm -rf /var/lib/apt/lists/*
 
+# Install Oracle JDKs
+RUN echo debconf shared/accepted-oracle-license-v1-1 select true | debconf-set-selections
```

Comment:
Is there a reason this first line is in its own RUN statement? I'd prefer to not add a separate image layer just for this.